components/QuestionAdd/OutputTypeField.tsx

```diff
@@ -1,32 +1,42 @@
+import { DmpIcon } from "@/components/Icons";
+
 import styles from './questionAdd.module.scss';
 
 const outputTypeOptions = [
   { id: 'defaults', name: 'Use defaults' },
-  { id: 'mine', name: 'Use mine' },
-  { id: 'addToDefaults', name: 'Add mine to defaults' }
+  { id: 'mine', name: 'Customize output list' },
 ];
 
 const defaultOutputTypes = [
-  'Audiovisual',
-  'Collection',
-  'Data paper',
-  'Dataset',
-  'Event',
-  'Image',
-  'Interactive resource',
-  'Model representation',
-  'Physical object',
-  'Service',
-  'Software',
-  'Sound',
-  'Text',
-  'Workflow'
+  { id: 'Audiovisual', type: 'Audiovisual', description: 'A series of visual representations imparting an impression of motion when shown in succession. May or may not include sound.' },
+  { id: 'Collection', type: 'Collection', description: 'An aggregation of resources, which may encompass collections of one resourceType as well as those of mixed types. A collection is described as a group; its parts may also be separately described.' },
+  { id: 'Data paper', type: 'Data paper', description: 'A factual and objective publication with a focused intent to identify and describe specific data, sets of data, or data collections to facilitate discoverability.' },
+  { id: 'Dataset', type: 'Dataset', description: 'Data encoded in a defined structure.' },
+  { id: 'Event', type: 'Event', description: 'A non-persistent, time-based occurrence.' },
+  { id: 'Image', type: 'Image', description: 'A visual representation other than text.' },
+  { id: 'Interactive resource', type: 'Interactive resource', description: 'A resource requiring interaction from the user to be understood, executed, or experienced.' },
+  { id: 'Model representation', type: 'Model representation', description: 'An abstract, conceptual, graphical, mathematical or visualization model that represents empirical objects, phenomena, or physical processes.' },
+  { id: 'Physical object', type: 'Physical object', description: 'A physical object or substance.' },
+  { id: 'Service', type: 'Service', description: 'An organized system of apparatus, appliances, staff, etc., for supplying some function(s) required by end users.' },
+  { id: 'Software', type: 'Software', description: 'A computer program other than a computational notebook, in either source code (text) or compiled form. Use this type for general software components supporting scholarly research. Use the “ComputationalNotebook” value for virtual notebooks.' },
+  { id: 'Sound', type: 'Sound', description: 'A resource primarily intended to be heard.' },
+  { id: 'Text', type: 'Text', description: 'A resource consisting primarily of words for reading that is not covered by any other textual resource type in this list.' },
+  { id: 'Workflow', type: 'Workflow', description: 'A structured series of steps which can be executed to produce a final outcome, allowing users a means to specify and enact their work in a more reproducible manner.' }
```

Comment:
Thanks for adding these. Do you think we should ultimately store them on the backend?